### DISCUSSION TITLE
I have a few questions about this

### DISCUSSION THREAD

**Discussion by shyam-appymonkeys** [code4game/glTFForUE4#59] [Q&A]
1. What part of the gltf file is used to make a skeleton/animation file in Unreal?
2. Is it possible to import an animated model without a skeleton?

Take this for example:
https://sketchfab.com/3d-models/low-poly-truck-car-drifter-f3750246b6564607afbefc61cb1683b1
If I download the gltf version and import it in, there is no skeleton or anim files.
From the same link, if I get the fbx file and import that in, now there are multiple skeleton and anim files.

I have my own custom made gltf files from an exporter that I'm making, and I need to be able to import it in Unreal(I have to add that the model that is being exported is not a skinned mesh, it does not have any skinning or weights on it, it just a bunch of points and objects parented to those points that are translated and rotated). At the moment, if I import in the custom file, it imports them all as separated static meshes. There is no skeleton or anim files.

**Comment by jixingcn** [comment]
In the low-poly_truck_car_drifter, all animations are represented for the scene, not for a mesh.
The glTF is scene format, so it should be a level when imported into UE. Now the plugin does not support importing as a level, and it imports the glTF as a mesh group. 😢 

I think that glTF should be converted to a blueprint that can do more complex animations if the plugin wants to implement more features of glTF. I need to make a plan for that.

By the way, the plugin supports two animation formats for mesh, Skeleton Mesh and Morph Mesh.

**Reply by shyam-appymonkeys** [reply]
How does the morph mesh work, what does it do different from skeleton mesh? Do I have to make any changes in the gltf file for it? Would the low-poly-car work with the morph mesh?

**Reply by shyam-appymonkeys** [reply]
Also, do you plan on updating the plugin to allow scene animations?

**Reply by shyam-appymonkeys** [reply]
https://github.com/rdeioris/glTFRuntime
This is capable of importing both the car and my custom file but it is only during play in editor. It does not currently allow regular imports into the content browser.
